pkg/meta/dump.go

```diff
@@ -405,6 +425,18 @@ func decodeEntry(dec *json.Decoder, parent Ino, cs *DumpedCounters, parents map[
 						}
 					}
 				}
+
+				p := parent
+				for {
+					if q := quotas[p]; q != nil {
```

Comment:
It's expensive to look for quota for every file, we could merge the stats for each directory first, then update the quota for each directory

Reply:
Done